### PR TITLE
Changeling Rapid Regen buff

### PR DESCRIPTION
**Pull request by adacovsk**
Play tested some ling spells yesterday, rapid regen just heals damage. Doesn't add back limbs, fix broken bones, remove negative effects, etc. so basically trash. Currently has a cost of 5 evolution points, making it the second-most expensive Changeling spell.

## What this does
- uses rejuvenate instead of just reducing damage, increases chem cost by 10. This regenerates limbs like Regenerative Stasis
- moves some lists to rejuvinate
- reduces the junk in Regenerative Stasis that's redundant

## Why it's good
Rapid Regen costs a lot and has little benefit, so it's best to either r*move it, or change it. So I buffed it.

:cl:
 * tweak: Changeling's Rapid Regen should work like it's description says

### CODE REVIEW

**Comment by bathosbathtime** [comment]
This also increases the chemical cost from 30 to 40, which I think is fine but I would note it in at least the op and maybe the change log. Also, is this the same rejuvenate that the admin button uses? It will automatically remove cuffs and other bindings if it is so I would check for that.

File: code/modules/spells/changeling/rapidregen.dm [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
-			C.adjustToxLoss(-10)
-			C.adjustOxyLoss(-10)
-			C.adjustFireLoss(-10)
-			sleep(10)
+/spell/changeling/rapidregen/cast_check(skipcharge = 0, var/mob/user = usr)
+	. = ..()
+	if (!.)
+		return FALSE
+	if(user.mind && user.mind.suiciding)			//no reviving from suicides
+		to_chat(user, "<span class='warning'>Why would we wish to regenerate if we have already committed suicide?")
```
Comment (gurfan) [inline code/modules/spells/changeling/rapidregen.dm]: missing closing tag

**Comment by adacovsk** [comment]
>Also, is this the same rejuvenate that the admin button uses? It will automatically remove cuffs and other bindings if it is so I would check for that.

you're actually talking about Revive, not rejuvenate

**Comment by bathosbathtime** [comment]
Oh, my mistake.

**Comment by Killette3** [comment]
I really like using rapid regeneration the way it currently is. It acts as a pretty effective emergency combat steroid for a fight where you could be taking sustained damage over time and not wanting a one time heal every time. How would you want to combine or translate these effects?

**Comment by adacovsk** [comment]
> I really like using rapid regeneration the way it currently is. It acts as a pretty effective emergency combat steroid for a fight where you could be taking sustained damage over time and not wanting a one time heal every time. How would you want to combine or translate these effects?

I mean, I could keep the slow damage recovery... It's not really rapid though?

**Comment by adacovsk** [comment]
okay updated it but upped the cooldown

**Comment by PrimeDSS13** [comment]
I think that the vampire coffin effect, if sped up, could serve fairly well for a "rapid regen".

**Comment by githubuser4141** [comment]
The 2.5 minute cooldown, along with the chem cost being raised and the healing effect being halved makes this feel like a nerf to its main role, acting like a quick stimpak/healing item. Now it feels like it's supposed to be a regen stasis 2.0 spell but doesn't really fit that role either?

**Comment by adacovsk** [comment]
> The 2.5 minute cooldown, along with the chem cost being raised and the healing effect being halved makes this feel like a nerf to its main role, acting like a quick stimpak/healing item. Now it feels like it's supposed to be a regen stasis 2.0 spell but doesn't really fit that role either?

Well the healing lasts for 100 seconds, so I will use 100 seconds instead of 2.5 minutes

**Pull request merged**